### PR TITLE
feat(lang): add json highlights

### PR DESCRIPTION
**Pull request by steelsojka**

### CODE REVIEW

File: queries/json/highlights.scm [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(true) @constant.builtin
+(false) @constant.builtin
```
Comment (kyazdani42) [inline queries/json/highlights.scm:2]: isn't there `@boolean` @vigoux  ?

Comment (steelsojka) [reply queries/json/highlights.scm:2]: Oops... don't know why I did constant...

Comment (steelsojka) [reply queries/json/highlights.scm]: @kyazdani42 Updated

**Comment by steelsojka** [comment]
> This highlighting might have issue due to the overlapping matches (`label` and `string`).

Should be fixed now. `@string` is only for the value of a pair instead just a string node.

**Pull request merged**